A few tips (#143)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc

{ProjectShort} uses ElasticSearch to store events, which requires a larger than normal `vm.max_map_count`. The `vm.max_map_count` value is set by default in {OpenShift}.

> TIP: If your host platform is a typical {OpenShift} 4 environment, it should not be necessary to make any adjustments. The default node tuning operator is configured to account for ElasticSearch workloads.

Reviewer: ```suggestion
TIP: If your host platform is a typical {OpenShift} 4 environment, do not make any adjustments. The default node tuning operator is configured to account for ElasticSearch workloads.
```

Removing modal verb
